library/ssl_12_gen.c

```diff
@@ -61,7345 +54,1073 @@
 #include "mbedtls/oid.h"
 #endif
 
-#if defined(MBEDTLS_SSL_PROTO_DTLS)
+/*
+ * HANDSHAKE STATE: Outgoing `Certificate`
```

Comment:
The primary goal of this PR is to split existing files in a minimal way so that messaging and logic functionality are separate. I'm in principle fine with further splitting that goes beyond that, but I'd prefer to have this done in a separate PR.